framework-api-vulnerabilities/activities/clean-up/deploy/Makefile

```diff
@@ -0,0 +1,20 @@
+FILE	:= ../flag
+FLAG	:= $(shell cat $(FILE))
+
+run: generate build
+	docker run -d -p 7004:80 --name sss-web-07_clean-up sss-web-07_clean-up
+
+build: generate
+	docker build -f Dockerfile -t sss-web-07_clean-up ..
+	rm ../src/data/users/user-information-expanded.json
```

Comment:
Does this action is necessary?
From what I saw the `run` rule depends on `generate` rule so always it would be generated that file with the flag.

Reply:
Should I place the line under the `run` rule, then?

Reply:
Or under `clean` might be an option